src/AppBundle/Controller/UserController.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace AppBundle\Controller;
+
+use AppBundle\Entity\User;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+
+class UserController extends Controller
+{
+    /**
+     * @Route("/users", name="users_list")
+     * @Template("@App/users.html.twig")
+     *
+     * @param Request $request
+     * @return array
+     */
+    public function usersListAction(Request $request)
+    {
+        $em = $this->getDoctrine()->getManager();
+        $users = $em->getRepository(User::class)->getUsersByParams($request->query);
+        if (!$users) {
```

Comment:
Do i need to move 
`$paginator = $this->get('knp_paginator');
        $pagination = $paginator->paginate($users, $request->query->getInt('page', 1), 10);`
to repo?